test/pjrt/test_runtime.py

```diff
@@ -14,6 +15,8 @@
 class TestExperimentalPjrt(parameterized.TestCase):
 
   def setUp(self):
+    global xr
+    reload(xr)
```

Comment:
why is this needed

Reply:
I added @run_once decorator for using_pjrt(), which remembers the previous call result. Without the reload, each test will impact with each other under the same test class.